src/utils/sdfg_meta_dict.json

```diff
@@ -1 +1,7274 @@
+{
```

Comment:
is there a script / github action to generate this file?

Reply:
Now there is, `scripts/generate_meta_dict.sh`